src/command_modules/azure-cli-batch/setup.py

```diff
@@ -28,8 +28,8 @@
 ]
 
 DEPENDENCIES = [
-    'azure-batch==1.1.0',
-    'azure-mgmt-batch==2.0.0',
+    'azure-batch==2.0.0',
+    'azure-mgmt-batch==3.0.0',
```

Comment:
You also depend on azure-mgmt-keyvault now as it's used in `_validators.py`.
Should add it here. Please make sure the version specified is the same as setup.py in Keyvault module.